.nvmrc

```diff
@@ -1 +1 @@
-lts/fermium
+18
```

Comment:
Curious, can we use `lts/hydrogen`?

Reply:
We can but I always forget what those codenames mean and realised we can just put the number in here. Do you have a preference the other way?

Reply:
Not really, flagged mostly because the old one and the public website use the codenames. I assume they won't differ in the required version?